### PR TITLE
Decreases the Brig Physician job slot back to one

### PR DESCRIPTION
**Pull request by Aurorablade**
:cl: Fethas
tweak: Removes sec access from the paramedic
/ :cl:

### CODE REVIEW

File: code/game/jobs/job/medical.dm [hunk 1]
```diff
@@ -250,7 +250,7 @@
 	is_medical = 1
 	supervisors = "the chief medical officer"
 	selection_color = "#ffeef0"
-	access = list(access_paramedic, access_medical, access_sec_doors, access_maint_tunnels, access_external_airlocks, access_morgue)
+	access = list(access_paramedic, access_medical, access_maint_tunnels, access_external_airlocks, access_morgue)
 	minimal_access=list(access_paramedic, access_medical, access_sec_doors, access_maint_tunnels, access_external_airlocks, access_morgue)
```
Comment (FlattestGuitar) [inline code/game/jobs/job/medical.dm]: `access_sec_doors` :eyes:

**Comment by SamHPurp** [comment]
Has this actually proved a problem? I've never been in a round where I've seen it be a problem. Not against doing it, though.

**Comment by taukausanake** [comment]
Why did it even have Sec access to start with? So it could get the dying from in there? I thought that was the whole reason Brig Physician existed, at least to help keep alive before needing Medical but then the normal phys can come over and be let in to take the body away

**Comment by Aurorablade** [comment]
its not been an issue since alot of people did not know it was a thing..but some did..like a recent round where a paramedic on a ambulance kept making hit and runs on sec with a syringe gun...

**Comment by Spacemanspark** [comment]
This was bad three years ago, it was removed.
...Then it came back again and continued to be bad. I fully support its removal.

**Comment by Allfd** [comment]
Wait....

When did this happen?

The paramedic had sec access?

**Comment by SamHPurp** [comment]
@Allfd as far back as I can remember, ya.

**Comment by Xantholne** [comment]
> Wait, when was the brig physician count raced to two?

> It took me a minute to realize it meant paramedic but my question still stands; because. Brig Physician.

**Comment by Fox-McCloud** [comment]
>This was bad three years ago, it was removed.
...Then it came back again and continued to be bad. I fully support its removal.

@Spacemanspark remember that weird white Tajaran? She played paramedic almost exclusively and abused the hell out of this access.

**Comment by Spacemanspark** [comment]
Katherine Black was terrible.

**Pull request merged**